packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -153,6 +155,10 @@ export interface VideoGalleryProps {
    * Camera control information for button to switch cameras.
    */
   localVideoCameraCycleButtonProps?: LocalVideoCameraCycleButtonProps;
+  /**
+   * Customize video rendering options
+   */
+  videoRenderingControls: VideoRenderingControlOptions;
```

Comment:
Any new props on something exported publicly must be optional so as to not be a breaking API change
```suggestion
  videoRenderingControls?: VideoRenderingControlOptions;
```

Reply:
We have decided to just remove videoRenderingControlOptions